<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotePad</class>
 <widget class="QMainWindow" name="NotePad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePad</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose210642"/>
    <addaction name="actionExit210642"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionSelectFont210642"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionAbout210642"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt210642"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="actionCut210642"/>
    <addaction name="actionCopy210642"/>
    <addaction name="actionPaste210642"/>
    <addaction name="actionUndo210642"/>
    <addaction name="actionRedo210642"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/new.png</normaloff>:/icons/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="toolTip">
    <string>Создать новый документ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClose210642">
   <property name="text">
    <string>Закрыть</string>
   </property>
   <property name="toolTip">
    <string>Закрыть текущее окно</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit210642">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Закрыть приложение</string>
   </property>
  </action>
  <action name="actionSelectFont210642">
   <property name="text">
    <string>Select Font</string>
   </property>
   <property name="toolTip">
    <string>Выберите отображаемый шрифт</string>
   </property>
  </action>
  <action name="actionAbout210642">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionAboutQt210642">
   <property name="text">
    <string>О Qt</string>
   </property>
  </action>
  <action name="actionCut210642">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/cut.png</normaloff>:/icons/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy210642">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/copy.png</normaloff>:/icons/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste210642">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/paste.png</normaloff>:/icons/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUndo210642">
   <property name="text">
    <string>Отменить</string>
   </property>
   <property name="toolTip">
    <string>Отменить последнее действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo210642">
   <property name="text">
    <string>Вернуть</string>
   </property>
   <property name="toolTip">
    <string>Вернуть последнее действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/open.png</normaloff>:/icons/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="toolTip">
    <string>Открыть документ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="toolTip">
    <string>Сохранить документ</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
   <property name="toolTip">
    <string>Сохранить документ как</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
